lib/check_params.h

```diff
@@ -593,11 +613,36 @@ void printQudaMultigridParam(QudaMultigridParam *param) {
     P(setup_maxiter_refresh[i], INVALID_INT);
 #endif
 
+#ifdef INIT_PARAM
+    P(setup_ca_basis[i], QUDA_INVALID_BASIS);
+    P(setup_ca_basis_size[i], 4);
+    P(setup_ca_lambda_min[i], 0.0);
+    P(setup_ca_lambda_max[i], -1.0);
+#else
+    P(setup_ca_basis[i], QUDA_INVALID_BASIS);
+    P(setup_ca_basis_size[i], INVALID_INT);
+    P(setup_ca_lambda_min[i], INVALID_DOUBLE);
+    P(setup_ca_lambda_max[i], INVALID_DOUBLE);
+#endif
+
+
```

Comment:
clang-format suggested the following change:


```
-
```

<!-- identifier: clang-format -->
<!-- File: lib/check_params.h -->